### ISSUE TITLE
GraphMLImport throws NullPointerException if property is applied at the <graphml> level

### ISSUE THREAD

**Issue by tschultze** [jgrapht/jgrapht#321]
A graphML format file created by yEd contains an attribute associated with the graph itself, present just before the closing </graphml> element. In this case the currentNodeOrEdge collection has no entries and currentNodeOrEdge.peek() returns a null.

**Comment by d-michail** [comment]
Could you provide a small example input?

**Comment by d-michail** [comment]
Could you test this [branch](https://github.com/d-michail/jgrapht/tree/graphml-graph-level-key-fix)?

Just clone it and install using 
```
git clone git@github.com:d-michail/jgrapht.git --branch graphml-graph-level-key-fix
cd jgrapht
mvn install
```
and change your pom.xml to use 1.0.1-SNAPSHOT as the version of the jgrapht-ext and jgrapht-core
```
<dependency>
    <groupId>org.jgrapht</groupId>
    <artifactId>jgrapht-ext</artifactId>
    <version>1.0.1-SNAPSHOT</version>
</dependency>
```
Thanks!

**Comment by tschultze** [comment]
I just tested this branch against my test case and it now handles that element created by the yEd editor.

**Comment by d-michail** [comment]
Thanks for reporting this! 

The fix will be merged in pull-request #322.

**Comment by jkinable** [comment]
Thanks for reporting!
We usually include a reference to the person who reported a bug or contributed code in the history file. For a proper attribution of this bug, could you give your full name? If you prefer not to give this, then I'll just use your github name "@tschultze".

**Comment by tschultze** [comment]
Tim Schultze

I'm working on reading in graphml files produced by the yEd graphical editor, particularly interested in shortest path calculations. Are you interested in issues related to yEd's XSD based elements. Currently, we're massaging out a lot of the yEd stuff because it causes the import to throw exceptions (mainly on yEd's extensions)?

**Comment by d-michail** [comment]
Hi Tim,

our support for GraphML allows only version 1.0 without any custom extensions. Using input which follows the [ygraphml](http://www.yworks.com/xml/schema/graphml/1.1/ygraphml.xsd) schema, will throw an exception.

While it is a good idea to support such extensions, we do not have the time. Of course contributions are always welcome :-) .

**Comment by d-michail** [comment]
Fixed in pull-request #322.
